controllers/loadtest_controller.go

```diff
@@ -153,7 +198,18 @@ func newDriverPod(loadtest *grpcv1.LoadTest, component *grpcv1.Component) (*core
 		return nil, err
 	}
 
-	addDriverPort(&pod.Spec.Containers[0])
+	spec := &pod.Spec
+
+	// TODO: Avoid referencing containers by index, use names
+	addDriverPort(&spec.Containers[0])
```

Comment:
Will follow up with another PR that contains function for looping and finding the appropriate container. This will be useful in multiple places in the future.